just_audio/ios/Classes/JustAudioPlayer.swift

```diff
@@ -71,48 +69,57 @@ public class JustAudioPlayer: NSObject {
             case "pause":
                 player.pause()
                 result([:])
+            case "stop":
+                player.stop()
+                result([:])
             case "setVolume":
                 player.setVolume(Float(request["volume"] as! Double))
                 result([:])
-//            case "setSkipSilence":
-//                print("TODO: setSkipSilence", request)
-//                result([:])
+            case "setPitch":
+                player.setPitch(Float(request["pitch"] as! Double))
+                result([:])
+            case "setSkipSilence":
+                player.setSkipSilence(request["enabled"] as! Bool)
+                //result(PluginError.notImplemented(call.method))
+                    result([:])
```

Comment:
Is it implemented or not?

Reply:
still not, I'll reset to the previous comment but the example app will not start

Reply:
let me know what do you prefer, leave like this or let the app crash